data-common.json

```diff
@@ -25,11 +25,13 @@
   },
   "firefox": {
     "developer": {
+      "versions": ["firefox52", "firefox53"],
```

Comment:
Sorry, I missed the earlier discussion. I think adding an array of channels, including "stable", to each version is a much less weird approach than duplicating the source of truth for versions.